src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/utils/InputParamsVerifier.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.controllers.utils;
+
+
+import uk.gov.hmcts.reform.wataskmanagementapi.exceptions.v2.InvalidRequestException;
+import uk.gov.hmcts.reform.wataskmanagementapi.services.utils.DeleteTaskConstants;
+
+import static java.util.regex.Pattern.compile;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+import static uk.gov.hmcts.reform.wataskmanagementapi.services.utils.DeleteTaskConstants.CASEID_DELETE_EXCEPTION_MESSAGE;
+
+public final class InputParamsVerifier {
+
+    private InputParamsVerifier() {
+    }
+
+    public static void verifyCaseId(final String caseRef) {
+        if (isEmpty(caseRef)
```

Comment:
As discussed, for a new line we need unit test coverage.  See ref -> https://sonarcloud.io/component_measures?metric=new_coverage&selected=uk.gov.hmcts.reform%3Awa-task-management-api%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fwataskmanagementapi%2Fcontrollers%2Futils%2FInputParamsVerifier.java&view=list&pullRequest=991&id=uk.gov.hmcts.reform%3Awa-task-management-api

Reply:
Done